pkg/lib/robots/robots_txt.go

```diff
@@ -0,0 +1,23 @@
+package robots
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+// Disallow all crawlers
+var robotsTxt = strings.Trim(`
+User-agent: *
+Disallow: /
+`, "\n")
+
+type Handler struct{}
+
+func (h *Handler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
+	body := []byte(robotsTxt)
+	rw.Header().Set("Content-Type", "text/plain")
+	rw.Header().Set("Content-Length", strconv.Itoa(len(body)))
+	rw.WriteHeader(http.StatusOK)
+	_, _ = rw.Write(body)
```

Comment:
Add a test for this handler.

Reply:
What do you want to test? The response body?

Reply:
Yes. The response body.

Reply:
It looks like testing the framework :sosoad:

Reply:
Originally the body was generated by strings.Trim, so I said we need to test it. If the body is a literal string, then the test is not needed.